### PR TITLE
Improvements to invoke commands for handling Sealed Secrets

### PR DESCRIPTION
**Pull request by joakimnordling**
- Allow easy sealing/unsealing of multiple envs or all envs at once.
- Added the `--only-changed` flag to `unseal-secrets` to avoid resealing all unchanged secrets.

### CODE REVIEW

File: devops/tasks.py [hunk 1]
```diff
@@ -281,13 +282,20 @@ def _should_ignore(path):
         logger.info(f"Keeping temporary kube merges in {merge_tmp}")
 
 
-def get_master_key(env: str) -> None:
+def get_master_key(env: str, use_existing=False) -> Path:
```
Comment (lietu) [inline devops/tasks.py:285]: I get you might want this to be `False` if you specifically issue a command to download the new one from CLI, but in other cases why would you default to `False`?

File: devops/tasks.py [hunk 2]
```diff
@@ -309,79 +317,124 @@ def get_master_key(env: str) -> None:
...
-    logger.info(f"Saving master key to {output_file}")
+    logger.info(f"Saving master key to {master_key_file}")
 
-    output_file.write_text(content, encoding="utf-8")
+    master_key_file.write_text(content, encoding="utf-8")
+    return master_key_file
 
 
-def unseal_secrets(env: str) -> None:
+def unseal_secrets(envs: Iterable[str]) -> None:
```
Comment (lietu) [inline devops/tasks.py:327]: Wouldn't it be easier and clearer if this iteration was done outside of this function in the few places it's needed in?

File: devops/tasks.py (hunk 1 above)
Comment (joakimnordling) [reply devops/tasks.py]: Really good point. Changed it the other way around.

File: devops/tasks.py (hunk 2 above)
Comment (joakimnordling) [reply devops/tasks.py]: True, changed that.

**Pull request merged**